decoupled-menu/src/main-menu.island.tsx

```diff
@@ -135,7 +214,6 @@ const Button = styled.button`
   margin: 0 0 -4px;
   box-shadow: none;
   flex-shrink: 0;
-  border-radius: 999px;
   transition: color 0.2s ease-in-out, background 0.2s ease-in-out, border 0.2s ease-in-out;
   width: 38px;
```

Comment:
@jdwjdwjdw This is being overriden below it by the width: auto

Reply:
Thanks - updated.